src/v3/src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,121 +10,70 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { useOdysseyDesignTokens } from '@okta/odyssey-react-mui';
-import {
-  Box,
-  Checkbox as CheckboxMui,
-  FormControl,
-  FormControlLabel,
-  FormHelperText,
-} from '@okta/odyssey-react-mui-legacy';
-import { Fragment, h } from 'preact';
+import { Checkbox as OdyCheckbox, CheckboxGroup } from '@okta/odyssey-react-mui';
+import { h } from 'preact';
 
 import { useWidgetContext } from '../../contexts';
-import { useAutoFocus, useValue } from '../../hooks';
+import { useAutoFocus, useHtmlContentParser, useValue } from '../../hooks';
 import {
   ChangeEvent,
   UISchemaElementComponent,
   UISchemaElementComponentWithValidationProps,
 } from '../../types';
-import { getTranslationInfo } from '../../util';
-import FieldLevelMessageContainer from '../FieldLevelMessageContainer';
+import { buildFieldLevelErrorMessages, getTranslationInfo, wrapInTranslateNo } from '../../util';
 import { withFormValidationState } from '../hocs';
 
 const Checkbox: UISchemaElementComponent<UISchemaElementComponentWithValidationProps> = ({
   uischema,
   errors,
   handleChange,
   handleBlur,
-  describedByIds,
 }) => {
   const value = useValue(uischema);
   const { loading } = useWidgetContext();
 
   const {
     options: { inputMeta: { name, mutable } },
     focus,
-    noTranslate,
     translations = [],
-    showAsterisk,
   } = uischema;
   const isReadOnly = mutable === false;
+  const checkboxId = `${name}-checkbox`;
   const labelInfo = getTranslationInfo(translations, 'label');
+  const label = useHtmlContentParser(labelInfo?.noTranslate
+    ? wrapInTranslateNo(labelInfo.value)
+    : labelInfo?.value) as string | undefined;
   const descriptionInfo = getTranslationInfo(translations, 'description');
+  const description = useHtmlContentParser(descriptionInfo?.noTranslate
+    ? wrapInTranslateNo(descriptionInfo.value)
+    : descriptionInfo?.value) as string | undefined;
```

Comment:
The granular consent views are some of the only ones where we have to be able to apply `translate="no"` to part of a component but not another (e.g. don't translate the checkbox label but do translate the description/hint below it). To get around this, I wrote this utility to wrap a string to not be translated and then use the HTML content parser hook in the `Checkbox` component to parse this HTML string to a React element. Fortunately, Odyssey does not actually strictly type check the type once it is passed in and will render a React element at runtime so casting the wrapped `label` and `description` is a (not so great) workaround for this issue

Reply:
Can we create a ticket to review `translate` prop later? 
Possibly changing shape of it from `yes | no` to map like `{ hint: 'yes' | 'no', label: 'yes' | 'no', root: 'yes' | 'no' }` and add support of it to Odyssey and SIW.

Reply:
Yeah, we can talk to the Odyssey team about this. This issue was something we knew was going to come up when we implemented this in Odyssey, but we didn't want to make the API too complex since we only have to do this workaround for the granular consent view in SIW